jenkins/Jenkinsfile

```diff
@@ -21,7 +21,7 @@ try{
     node("LinuxSlave") {
         stage("Testing revisions with Artifactory"){
             checkout scm
-            for (arti_version in ["6.9.0", "6.10.6"]){  // TODO: Add latest!
```

Comment:
I think I would keep a relatively "old" Artifactory version too, 6.9.0 for example, just in case. Is it possible with this new code?

Reply:
Yes, the same implementation works with `6.9.0`: https://conan-ci.jfrog.info/job/Preflight_develop/573/consoleFull